modules/benchmarks/arrow-benchmarks-fx/build.gradle

```diff
@@ -23,24 +23,25 @@ apply plugin: "io.morethan.jmhreport"
 
 jmh {
     include = [
-            'arrow.benchmarks.Async',
-            'arrow.benchmarks.AttemptNonRaised',
-            'arrow.benchmarks.AttemptRaisedError',
+//            'arrow.benchmarks.Async',
```

Comment:
Should these be reenabled?

Reply:
We're currently not using these atm in CI with `Hood` so this does not impact anything. I can revert it.

Reply:
I'd say remove, don't delete.